.github/CODEOWNERS

```diff
@@ -24,10 +24,13 @@
 /modular_skyrat/modules/reagent_forging/ @jjpark-kb
 /modular_skyrat/modules/xenoarch/ @jjpark-kb
 
-# Maptainers
-/_maps/ @KathrinBailey
-
 # Zonepace
 /modular_skyrat/master_files/code/modules/client/preferences/headshot.dm @Zonespace27
 /modular_skyrat/modules/contractor/ @Zonespace27
 /modular_skyrat/modules/gun_cargo/ @Zonespace27
+
+# Multiple Owners
+
+# Maptainers
+/_maps/ @Jolly-66
+/_maps/ @KathrinBailey
```

Comment:
thats not how you do this

Reply:
What do you mean? What are you specifically commenting on?

![image](https://user-images.githubusercontent.com/70232195/175529007-186334e4-a60c-4b40-8d44-3ac8a4abec73.png)